code/go/0chain.net/blobber/node.go

```diff
@@ -30,7 +30,12 @@ func setupNode() error {
 		logging.Logger.Info("self identity", zap.Any("id", node.Self.ID))
 	}
 
-	node.Self.SetHostURL(hostname, httpPort)
+	if httpsPort > 0 {
```

Comment:
How are we populating this value?

Reply:
yes. it is populated on flag.go, and cert is checked on http.go.